pkg/apis/middleware/session.go

```diff
@@ -40,7 +41,7 @@ func CreateTokenToSessionFunc(verify VerifyFunc) TokenToSessionFunc {
 		}
 
 		if claims.Verified != nil && !*claims.Verified {
-			return nil, fmt.Errorf("email in id_token (%s) isn't verified", claims.Email)
+			return nil, fmt.Errorf("email in id_token (%s) isn't verified", logger.WrapSensData(claims.Email))
```

Comment:
If we align on the `scrub` terminology -- this might be a cleaner helper logger method as `logger.Scrub(string) string`